File: loopr-ios/Models/News/IndexAction.swift
//
//  CancelType.swift
//  loopr-ios
//
//  Created by kenshin on 2018/6/13.
//  Copyright © 2018年 Loopring. All rights reserved.
//

import Foundation

enum IndexAction: Int, CustomStringConvertible {
    
    case confirm = 1
    case cancel = -1
    
    var description: String {
        switch self {
        case .confirm: return "confirm"
        case .cancel: return "cancel"
        }
    }
}

File: android/walletsdk/src/main/java/leaf/prod/walletsdk/model/news/IndexAction.java
/**
 * Created with IntelliJ IDEA.
 * User: kenshin wangchen@loopring.org
 * Time: 2018-12-24 4:10 PM
 * Cooperation: loopring.org 路印协议基金会
 */
package leaf.prod.walletsdk.model.news;

public enum IndexAction {

    CONFIRM(1), CANCEL(-1);

    private final int value;

    IndexAction(int value) {
        this.value = value;
    }

    public int getValue() {
        return value;
    }
}
